sdk/core/Azure.Core/src/ResponseClassifier.cs

```diff
@@ -16,7 +16,14 @@ public class ResponseClassifier
         /// </summary>
         public virtual bool IsRetriableResponse(HttpMessage message)
         {
-            return message.Response.Status == 429 || message.Response.Status == 503;
+            var responseStatus = message.Response.Status;
+
+            return responseStatus == 408 ||
```

Comment:
Would a switch be faster here? Or even comparison logic like other languages, though I doubt that (seems more convenience in code done that way).